packages/zoe/test/unitTests/zcf/test-zcf.js

```diff
@@ -302,7 +303,7 @@ test(`zcf.makeInvitation - no customProperties`, async t => {
   // https://github.com/Agoric/agoric-sdk/issues/1705
   // @ts-ignore
   const details = await E(zoe).getInvitationDetails(invitationP);
-  t.deepEqual(details, {
+  assertAmountsEqual(t, details, {
```

Comment:
Oh no, this is passing but it shouldn't. This is a record, not an amount, so when `assertAmountsEqual` checks `.brand` and `.value` both are undefined, and undefined === undefined so this passes.

Reply:
Also, even if these were amounts, we should ensure this fails because MathKind.SET wasn't passed in, so MathKind.NAT was used as the default, and that is wrong.

Reply:
I reverted this incorrect conversion fo `deepEqual()` to `assertAmountsEqual()`. I'll convert the PR to `draft` until we have a solution for AmountMaths.

blocked on: #2311